platforms/gke/base/use-cases/federated-learning/examples/nvflare-tff/README.md

```diff
@@ -0,0 +1,281 @@
+
+   ```bash
+   nvflare provision
+   ```
+
+   The provisioning tool generates deployment descriptors:
+
+   - `server1` is the server that will aggregate all the results from the
+     computation
+   - `site-1` and `site-2` are the clients that will be connected to the server
+   - `admin@nvidia.com` is the administration client to start and list jobs
+
+1. Exit the NVIDIA FLARE container:
+
+   ```bash
+   exit
+   ```
+
+1. Copy the workspace folder to Cloud Storage:
+
+   ```bash
+   gcloud storage cp --recursive "${HOME}/nvflare-workspace/workspace" "gs://${NVFLARE_EXAMPLE_WORKSPACE_BUCKET_NAME}"
+   ```
+
+### Push the example container image
+
+In this section, you run a script that pushes the container images to the
+Artifact Registry repository that the Federated learning reference architecture
+provides.
+
+To run the script, do the following:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Run the script to build the container image:
+
+   ```sh
+   "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/examples/nvflare-tff/push-container-image.sh"
+   ```
+
+### Verify that NVIDIA FLARE pods are running
+
+In this section, you verify that you deployed NVIDIA FLARE in the reference
+architecture:
+
+1. Open the
+   [GKE Workloads Dashboard](https://cloud.google.com/kubernetes-engine/docs/concepts/dashboards#workloads)
+   and verify that NVIDIA FLARE pods are running in the `fl-1` namespace. The
+   output to look for is similar to the following:
+
+   ```bash
+   Name                               Status Pods
+   nvflare-client1-57d5b45d84-bmv58   OK     1/1
+   nvflare-client2-895b65d8f-p4fs9    OK     1/1
+   nvflare-server1-66c44ddb47-dhtqz   OK     1/1
+   ```
+
+### Submit the training job
+
+In this section, you submit and run a training job:
+
+1. Open [Cloud Shell](https://cloud.google.com/shell).
+
+1. Configure `kubectl` to access the cluster:
+
+   ```bash
+   gcloud container clusters get-credentials <CLUSTER_NAME> --region <CLUSTER_REGION> --project <CLUSTER_PROJECT> --dns-endpoint
+   ```
+
+   Where:
+
+   - `<CLUSTER_NAME>` is the name of the GKE cluster.
+   - `<CLUSTER_REGION>` is the region of the GKE cluster.
+   - `<CLUSTER_PROJECT>` is the project of the GKE cluster.
+
+1. Get the list of NVIDIA FLARE pods running in the cluster:
+
+   ```bash
+   kubectl get pods --namespace fl-1
+   ```
+
+   The output is similar to the following:
+
+   ```bash
+   NAME                               READY   STATUS             RESTARTS        AGE
+   nvflare-client1-7c5c469cdb-s4xn9   3/3     Running            9 (4m38s ago)   33m
+   nvflare-client2-7b49bdd4d-h9xp5    3/3     Running            9 (4m56s ago)   66m
+   nvflare-server1-9f7cb8dfd-rzl27    3/3     Running            11 (42s ago)    66m
+   ```
+
+1. Open a shell in the NVIDIA FLARE server pod:
+
+   ```bash
+   kubectl exec --stdin --tty --namespace fl-1 <NVIDIA_FLARE_SERVER_POD_NAME> -- /bin/bash
```

Comment:
It might be worth adding a command to pull and set the pod name for simplicity. Something like:

`NVIDIA_FLARE_SERVER_POD_NAME=$(kubectl --namespace fl-1 ...) `

Reply:
keeping this simple because we don't currently reuse the pod name across the example.